perun-web-gui/src/main/java/cz/metacentrum/perun/webgui/client/UiElements.java

```diff
@@ -336,6 +336,7 @@ public void onClick(ClickEvent arg0) {
 		} else {
 			conf = new Confirm(header + ((!error.getErrorId().equals("")) ? " (" + error.getErrorId() + ")" : ""), layout, okClickHandler, reportClickHandler, okLabel, reportLabel, true);
 			conf.setCancelIcon(SmallIcons.INSTANCE.emailIcon());
+			conf.setOkIcon(SmallIcons.INSTANCE.cancelIcon());
```

Comment:
Hi, we use "stopIcon" instead of "cancelIcon" for generic "cancel" button action. Please change it here. Otherwise it looks great and can be merged afterwards. Thanks.